File: src/main/java/com/team3/gudit/auth/dto/AccessTokenResponseDto.java
package com.team3.gudit.auth.dto;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Builder
@JsonInclude(JsonInclude.Include.NON_NULL)
public class AccessTokenResponseDto {
    private String accessToken;
    private String tokenType = "Bearer";
    private Long expiresIn;
}

File: src/main/kotlin/com/team3/gudit/auth/dto/AccessTokenResponseDto.kt
package com.team3.gudit.auth.dto

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class AccessTokenResponseDto(
    val accessToken: String? = null,
    val tokenType: String? = null,
    val expiresIn: Long? = null,
)
